phpunit.xml.dist

```diff
@@ -20,10 +20,4 @@
         <ini name="date.timezone" value="Europe/London"/>
         <ini name="error_reporting" value="E_ALL"/>
     </php>
-    <logging>
```

Comment:
Some options here are depreacted and I believe no one use the reports so probably better to just remove them